docs/src/pages/docs/monoids/All.md

```diff
@@ -99,7 +99,11 @@ All(false).concat(All(false)) //=> All false
 All a ~> b -> Boolean
 ```
 
-Used to compare the underlying values of (2) `All` instances for equality by value, equals takes any given argument and returns `true` if the passed argument is a `All` with an underlying value equal to the underlying value of the `All` the method is being called on. If the passed argument is not a `All` or the underlying values are not equal, equals will return `false`.
+Used to compare the underlying values of (2) `All` instances for equality by
+value, equals takes any given argument and returns `true` if the passed argument
+is a `All` with an underlying value equal to the underlying value of the `All` the
```

Comment:
Maybe a little typo here `is an` instead of `is a` (Not a native language, I can be wrong)

Reply:
Ohhh. thought I got all of those. Good find.